src/meshweb/static/admin/map.js

```diff
@@ -162,301 +186,145 @@ async function getNewSelectedNodes(){
+    // Disable onLoad for Admin Panel while we navigate to a new page
+    dontListenForAdminPanelLoad();
+
+    if (installResponse.ok){
+        const installJson = await installResponse.json();
+        if (installJson.node && installJson.node.network_number)  {
+            admin_panel_iframe.src = `/admin/meshapi/node/${installJson.node.id}/change`;
         } else {
-            window.history.pushState(null, '', newUrl);
+            admin_panel_iframe.src = `/admin/meshapi/install/${installJson.id}/change`;
         }
-    }
-
-    const current_map = document.getElementById("map");
-
-    const text = await response.text();
-    const parser = new DOMParser();
-    const doc = parser.parseFromString(text, "text/html");
-
-    doc.getElementById("map").replaceWith(current_map);
-
-    // Keep the elements Google Maps injected in the header, otherwise the map breaks
-    const headerElementsToKeep = [];
-    for (const el of document.getElementsByTagName("head")[0].getElementsByTagName("script")){
-        if (el.src && el.src.startsWith("https://maps.googleapis.com/")) headerElementsToKeep.push(el);
-    }
-    for (const el of document.getElementsByTagName("head")[0].getElementsByTagName("style")){
-        if (el.textContent.indexOf(".gm") !== -1) headerElementsToKeep.push(el);
-    }
-
-    for (const el of headerElementsToKeep){
-        doc.getElementsByTagName("head")[0].appendChild(el);
-    }
-
-    // Replace the whole page with the new one
-    const newHTML = doc.getElementsByTagName("html")[0];
-    document.getElementsByTagName("html")[0].replaceWith(newHTML);
-
-    setMapProportions(currentSplit);
-
-    // Re-run other javascript to make page happy
-    if (window.DateTimeShortcuts) window.removeEventListener('load', window.DateTimeShortcuts.init);
-
-    const scriptsToReload = [];
-    for (const script of document.head.querySelectorAll('script')){
-        if (!script.src || !script.src.startsWith("https://maps.googleapis.com/")) scriptsToReload.push(script);
-    }
-    await loadScripts(scriptsToReload, document.head);
-
-    dispatchEvent(new Event('load'));
-}
-
-
-function shouldNotIntercept(target) {
-    const isForm = target.tagName === "FORM";
-    let url = ""
-    if (isForm){
-        url = new URL(target.action);
     } else {
-        url = new URL(target.href);
+        if (nodeResponse.ok)  {
+            const nodeJson = await nodeResponse.json();
+            admin_panel_iframe.src = `/admin/meshapi/node/${nodeJson.id}/change`;
+        }
     }
 
-    if (target.className === "capture-exclude") return true;
-    if (!url.pathname.startsWith("/admin/")) return true;
-    if (url.pathname.startsWith("/admin/login")) return true;
-    if (url.pathname.startsWith("/admin/logout")) return true;
-    if (url.pathname.endsWith("/export/") && isForm) return true;
-    if (url.host !== location.host) return true;
+    console.log(`Admin Panel src set to: ${admin_panel_iframe.src}`);
```

Comment:
Might want to gate all these log lines with a debug statement or something for cleanliness.